R/format-report.R

```diff
@@ -790,26 +837,32 @@ format_appendix <- function(extra_notes_data, return_vals = FALSE){
     # R CMD Check
     cat(sub_header_strs[1])
     cat_verbatim(check_output)
+
+    if (is.null(cov_results_flex)) {
```

Comment:
This should have been `cov_results_df`.  Using `cov_results_flex` makes the downstream handling of covr errors unreachable.

Will push a fixup, along with a test that enters the covr error arm below (i.e. would have flagged this issue).